frame/support/src/lib.rs

```diff
@@ -1381,15 +1381,23 @@ pub mod pallet_prelude {
 /// alias then the generation of the getter might fail. In this case getter can be implemented
 /// manually.
```

Comment:
```suggestion
/// NOTE: If the querykind generic parameter is still generic at this stage or is using some type
/// alias then the generation of the getter might fail. In this case the getter can be implemented
/// manually.
```

(cannot make suggestions directly for untouched parts of the doc)

Reply:
Also my brain definitely flagged "querykind" as a neologism that looks a little odd, consider styling it `QueryKind` or whatever the type name is.

Reply:
yes done in https://github.com/paritytech/substrate/pull/7922/commits/cf03d78a8d568758e409b21fe99e609faf4288c5